src/test/features/dashboard/routes/claimant.ts

```diff
@@ -50,7 +50,7 @@ describe('Dashboard - claimant page', () => {
           await request(app)
             .get(claimantPage)
             .set('Cookie', `${cookieName}=ABC`)
-            .expect(res => expect(res).to.be.successful.withText('Claim number', 'The defendant'))
+            .expect(res => expect(res).to.be.successful.withText('Claim number', 'Request a County Court Judgment'))
```

Comment:
Maybe just `withText('Claim number: ' + claimNumber)` would be enough here. A `Request a County Court Judgment` is very much depends on the state of the claim

Reply:
OK, I'll change it. I just fixed the test that was broken when we started displaying more specific status. The previous one was VERY generic ;)

Reply:
done